_layouts/news-grid.html

```diff
@@ -0,0 +1,63 @@
+<!doctype html>
+
+{% include head.html %}
+
+<body>
+    {% include header-extended.html %}
+
+    <main id="main-content">
+       
+      <section class="usa-card__header bg-blue-80v">
+            <div class="grid-container">
+                <h1 class="font-heading-3xl text-accent-cool">{{ page.title }}</h1>
+                <h2 class="font-alt-xl text-base-lightest tablet:grid-col-9">{{ page.description }}</h2>
+            </div>
+        </div>
+        </section>
+       
+ <section class="grid-container margin-top-4 margin-bottom-4 border-bottom border-base-light">
+    <p class="usa-intro mobile:font-heading-lg tablet:grid-col-9">
+		{{ page.intro }}
+	</p>
+ </section>
+ 
+ <section class="grid-container padding-y-2">
+  <div class="grid-row grid-gap">
+    <ul class="usa-card-group">
+      {% for article in page.articles %}
+ <li class="tablet:grid-col-4 usa-card">
+    <div class="usa-card__container">
+      <header class="usa-card__header">
+        <h6 class="text-base-dark">{{ article.source }}</h6>
```

Comment:
Similar issue to subtitle above but for additional reason: headings should follow their hierarchical order (2s after 1s, 3s after 2s, etc.). So we don't want to have an h6 followed by an h2. This is a common problem use case, where you have this kind of subtitle above the main title. I'll see if I can find a discussion of the preferred approach, but you can't go wrong with a paragraph and then use classes for styling to get the appearance you want.

Reply:
Okay thanks for explaining! This leads to some q's I have about using text styles in Figma and what happens during the handoff to developers. Definitely something we can meet and discuss because I think I could be more clear asking about it verbally, but I wanted to record it here so I don't forget.